src/vmm/src/lib.rs

```diff
@@ -301,6 +313,36 @@ impl VMM {
         Ok(())
     }
 
+    fn add_block_device(&mut self, cfg: &BlockConfig) -> Result<()> {
```

Comment:
Can you add unit tests for this code? It should be relatively straightforward to use the `mock_vmm`, and check the valid scenario: path is valid and exists, and at least one invalid scenario so that we check that the appropriate value is returned.

Reply:
I'd like to add this kind of testing in the following PR where I do some refactoring of the common bits of code between devices (i.e. block and net at that point)